phys/module_firebrand_spotting.F

```diff
@@ -1503,11 +1507,12 @@ SUBROUTINE firebrand_spotting_em_driver( &
 
         USE module_domain,            ONLY: domain_get_time_since_sim_start, &
                                             domain_clock_get, is_alarm_tstep
-        USE module_dm,                ONLY: wrf_dm_max_real, wrf_dm_sum_integer, local_communicator
+!        USE module_dm,                ONLY: wrf_dm_max_real, wrf_dm_sum_integer, local_communicator
```

Comment:
@mefrediani 
Maria,
If this "USE module_dm" is not required, just remove it instead of commenting it out.